wp-config.php

```diff
@@ -77,7 +77,7 @@
  *
  * @link https://codex.wordpress.org/Debugging_in_WordPress
  */
-define('WP_DEBUG', false);
```

Comment:
So pretty much if this is on, it should show me an error. Is there anyway to define this by environment?